static/src/javascripts/projects/common/modules/commercial/badges.js

```diff
@@ -28,12 +28,12 @@ define([
     var badgesConfig = {
             sponsoredfeatures: {
                 count:      0,
-                header:     config.switches.newCommercialContent ? 'Supported by:' : 'Sponsored by:',
+                header:     'Supported by:',
                 namePrefix: 'sp'
             },
             'advertisement-features': {
                 count:      0,
-                header:     config.switches.newCommercialContent ? 'Paid for by' : 'Brought to you by:',
+                header:     'Paid for by:',
```

Comment:
Should the colon also be removed for `foundation-features`? I added the colon in to `advertisement-features` to make it consistent with the `foundation-features` as well as to pass the test that was already there (but was essentially hidden behind the switch). More than happy to remove it, but would be good to see a spec that confirms the exact copy to be used.